README.md

```diff
@@ -13,23 +15,29 @@ the [WiPy](https://github.com/micropython/micropython/tree/master/cc3200).
 Put the modules ``mfrc522.py``, ``examples/read.py``, ``examples/write.py`` to the root of the flash FS on your board. 
 For the ESP8266 there are multiple solutions to do that. E.g. use the 
 [WebREPL file transfer](https://github.com/micropython/webrepl), or [mpfshell](https://github.com/wendlers/mpfshell). 
- 
+
+For ESP32 I use ampy to transfer the files.
+
 I used the following pins for my setup:
 
-| Signal    | GPIO ESP8266 | GPIO WiPy      | Note                                 |
+| Signal    | GPIO ESP8266 | GPIO WiPy      | ESP32                                |
 | --------- | ------------ | -------------- | ------------------------------------ |
-| sck       | 0            | "GP14"         |                                      |
-| mosi      | 2            | "GP16"         |                                      |
-| miso      | 4            | "GP15"         |                                      |
-| rst       | 5            | "GP22"         |                                      |
-| cs        | 14           | "GP14"         |Labeled SDA on most RFID-RC522 boards |
+| sck       | 0            | "GP14"         | 18                                   |
+| mosi      | 2            | "GP16"         | 23                                   |
+| miso      | 4            | "GP15"         | 19                                   |
+| rst       | 5            | "GP22"         | 4                                    |
+| cs        | 14           | "GP14"         | 2                                    |
```

Comment:
i don't have some of these pins on my wems oled esp32 devboard. 16, 5, 26, 25, 4 worked for me.